test/testInteraction.cpp

```diff
@@ -839,17 +839,17 @@ TEST(EMDoublePairProduction, limitNextStep) {
 
 TEST(EMDoublePairProduction, secondaries) {
 	// Test if secondaries are correctly produced.
-	ref_ptr<PhotonField> CMB_instance = new CMB();
-	ref_ptr<PhotonField> IRB = new IRB_Gilmore12();
-	ref_ptr<PhotonField> URB = new URB_Nitu21();
-	EMDoublePairProduction m(CMB_instance);
+	ref_ptr<PhotonField> cmb = new CMB();
+	ref_ptr<PhotonField> irb = new IRB_Gilmore12();
+	ref_ptr<PhotonField> urb = new URB_Protheroe96();
```

Comment:
Why is the URB model changed. If we want to test the URB_Protheroe as well I would suggest to add it as a new target.

Reply:
This was a copy-paste from before. I didn't pay attention to which background that was. 
I will revert it back to Nitu21.